src/NLog/Layouts/LayoutParser.cs

```diff
@@ -386,7 +386,10 @@ private static LayoutRenderer ParseLayoutRenderer(ConfigurationItemFactory confi
                         var value = ParseParameterValue(stringReader);
                         if (!string.IsNullOrEmpty(parameterName) || !StringHelpers.IsNullOrWhiteSpace(value))
                         {
-                            // TODO NLog 5.0 Should throw exception when invalid configuration (Check throwConfigExceptions)
+                            if (throwConfigExceptions ?? LogManager.ThrowConfigExceptions ?? LogManager.ThrowExceptions)
```

Comment:
Should also use:

```c#
var configException = new NLogConfigurationException($"Unrecognized property '{parameterName}={value}` for ${{{name}}} ({layoutRenderer?.GetType()})");
if (throwConfigExceptions ?? configException.MustBeRethrown())
     throw configException;
```

Reply:
Alright. I didn't change it since the messages differed: `Skipping unrecognized property`

Reply:
Maybe just repeat the trick you did here:

`InternalLogger.Error(ex, "{0} will be ignored.", configException.Message);`